e2e-tests/ukef/cypress/e2e/journeys/portal/amendments/pages/tfm/tfm-display-amendments-sorted-by-reference-number.spec.js

```diff
@@ -0,0 +1,140 @@
+import { getFormattedMonetaryValue, CURRENCY } from '@ukef/dtfs2-common';
+import { twoYears, threeYears, today, DD_MMMM_YYYY_FORMAT } from '@ukef/dtfs2-common/test-helpers';
+import { format } from 'date-fns';
+import { UNDERWRITER_MANAGER_DECISIONS, PIM_USER_1, TFM_URL } from '../../../../../../../../e2e-fixtures';
+import MOCK_USERS from '../../../../../../../../e2e-fixtures/portal-users.fixture';
+import { MOCK_APPLICATION_AIN_DRAFT } from '../../../../../../../../e2e-fixtures/gef/mocks/mock-deals';
+import { anIssuedCashFacility } from '../../../../../../../../e2e-fixtures/mock-gef-facilities';
+import facilityPage from '../../../../../../../../tfm/cypress/e2e/pages/facilityPage';
+import amendmentsPage from '../../../../../../../../tfm/cypress/e2e/pages/amendments/amendmentsPage';
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+const mockFacility = anIssuedCashFacility({ facilityEndDateEnabled: true });
+
+const CHANGED_FACILITY_VALUE = '20000';
+const CHANGED_FACILITY_VALUE_2 = '30000';
+const CHANGED_FACILITY_VALUE_3 = '40000';
+
+const submittedByString = `${BANK1_MAKER1.firstname} ${BANK1_MAKER1.surname} - ${BANK1_MAKER1.bank.name}`;
+
+context(
+  'Amendments - TFM - Amendments details page - TFM should display portal amendments and tfm amendments on the amendment details page in correct order',
```

Comment:
Looks like it's descending order, can we add this to the title?

```suggestion
  'Amendments - TFM - Amendments details page - TFM should display portal amendments and tfm amendments on the amendment details page in descending order',
```